sdk/loadtestservice/azure-mgmt-loadtesting/tests/test_resourceoperations.py

```diff
@@ -0,0 +1,161 @@
+        assert loadtest_resource_get.name == resource_group.name+"resource"
+        assert loadtest_resource_get.location == self.AZURE_LOCATION
+        assert loadtest_resource_get.identity != None
+        assert loadtest_resource_get.identity.type == "SystemAssigned, UserAssigned"
+        assert len(loadtest_resource_get.identity.user_assigned_identities) == 2
+        assert loadtest_resource_get.encryption != None
+        assert loadtest_resource_get.encryption.key_url == self.key.id
+        assert loadtest_resource_get.encryption.identity != None
+        assert loadtest_resource_get.encryption.identity.type == "UserAssigned"
+        assert loadtest_resource_get.encryption.identity.resource_id == self.msi1.id
+
+
+        # Update the load test resource
+        loadtestresourcePatchdata = LoadTestResourcePatchRequestBody(
+            identity=ManagedServiceIdentity(
+                type="SystemAssigned,UserAssigned",
+                user_assigned_identities={
+                    self.msi2.id: None
+                }
+            ),
+        )
+
+        # load test resource update begin - returns a poller
+        loadtest_resource_patch_poller = self.loadtestservice_client.load_tests.begin_update(resource_group.name, resource_group.name+"resource", loadtestresourcePatchdata)
+        
+        # Wait for the poller to complete
+        while loadtest_resource_poller.done()==False:
+            loadtest_resource_poller.wait(2)
+        
+        # Get the result of the poller
+        loadtest_resource_patch_response = loadtest_resource_patch_poller.result()
+
+        # Assert the result of the poller
+        assert loadtest_resource_patch_response != None
+        assert loadtest_resource_patch_response.id != None
+        assert loadtest_resource_patch_response.name == resource_group.name+"resource"
+        assert loadtest_resource_patch_response.location == self.AZURE_LOCATION
+        assert loadtest_resource_patch_response.identity != None
+        assert loadtest_resource_patch_response.identity.type == "SystemAssigned, UserAssigned"
+        assert len(loadtest_resource_patch_response.identity.user_assigned_identities) == 1
+        assert loadtest_resource_patch_response.encryption != None
+        assert loadtest_resource_patch_response.encryption.key_url == self.key.id
+        assert loadtest_resource_patch_response.encryption.identity != None
+        assert loadtest_resource_patch_response.encryption.identity.type == "UserAssigned"
+        assert loadtest_resource_patch_response.encryption.identity.resource_id == self.msi1.id
+
+        # Delete the load test resource - returns a poller
+        loadtest_resource_delete_poller = self.loadtestservice_client.load_tests.begin_delete(resource_group.name, resource_group.name+"resource")
+
+        # Wait for the deletion LRO to complete
+        while loadtest_resource_delete_poller.done()==False:
+            loadtest_resource_delete_poller.wait(2)
```

Comment:
```suggestion
        self.loadtestservice_client.load_tests.begin_delete(resource_group.name, resource_group.name+"resource").result()
```

same for similar operation. In mgmt, `.result()` means that operation will wait for polling over